src/js/utils/ValidatorUtil.js

```diff
@@ -1,10 +1,33 @@
 var ValidatorUtil = {
+  isDefined: function (value) {
+    return value != null && value !== '' || typeof value === 'number';
+  },
+
   isEmail: function (email) {
     return email != null &&
       email.length > 0 &&
       !/\s/.test(email) &&
       /.+@.+\..+/
       .test(email);
+  },
+
+  isInteger: function (value) {
+    const number = parseFloat(value);
+
+    return Number.isInteger(number);
+  },
+
+  isNumber: function (value) {
+    const number = parseFloat(value);
+
+    return !Number.isNaN(number) && Number.isFinite(number);
+  },
+
+  isNumberInRange: function (value, range = {}) {
```

Comment:
Yeah, you are right. That's over-engineering a bit. I think even `isNumber(x) && (x > 0)` could do the trick.